src/Aspire.Hosting.Azure/AzureWebPubSubResource.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Azure;
+
+namespace Aspire.Hosting.ApplicationModel;
+
+/// <summary>
+/// Represents an Azure Web PubSub resource.
+/// </summary>
+/// <param name="name">The name of the resource.</param>
+public class AzureWebPubSubResource(string name) :
+    AzureBicepResource(name, templateResouceName: "Aspire.Hosting.Azure.Bicep.webpubsub.bicep"),
+    IResourceWithConnectionString
+{
+    /// <summary>
+    /// Gets the "connectionString" output reference from the bicep template for Azure Web PubSub.
+    /// </summary>
+    public BicepOutputReference Endpoint => new("endpoint", this);
```

Comment:
Can you add a manifest test, like the following:

https://github.com/dotnet/aspire/blob/4a423f740cacbdee748380b9150d811157b3d8d5/tests/Aspire.Hosting.Tests/Azure/AzureBicepResourceTests.cs#L502-L529